test/chains/networkRequestChain.test.js

```diff
@@ -119,6 +165,31 @@ describe('Network request chain', () => {
 				},
 			},
 		});
+		assert.deepStrictEqual(toJSON({
+			comparator: {
+				type: SUBJ_COMPARATOR.EQUAL,
+				val: 'test',
+			},
+			wasMade: true,
+			isNegated: true,
+		}), {
+			type: 'eval',
+			request: {
+				type: 'wait',
+				timeout: 2000,
+				condition: {
+					subject: {
+						type: 'network',
+						requestInfo: [],
+						responseInfo: [],
+						compare: '!=',
+						val: 'test',
+					},
+					type: 'made',
```

Comment:
I'm not sure `subject.compare` is the right place to apply negation. Shouldn't it be applied to `type`? I.e. `!made`.

Reply:
@polomsky could you confirm which option is correct?

Reply:
you are right, only `made` and `!made`, `!=` can be part of `requestInfo` or `responseInfo`